beacon-chain/core/helpers/committee.go

```diff
@@ -412,6 +413,14 @@ func IsCurrentEpochSyncCommittee(
 		if err != nil {
 			return false, err
 		}
+
+		// Fill in the cache on miss.
+		go func() {
```

Comment:
why is this another go-routine

Reply:
What do you mean by another?

Reply:
Any reason to spin this off into a new go-routine rather than keeping it on the same thread. Is filling up the cache that expensive ?

Reply:
This should be returned as soon as possible given the 1/3 and 2/3 into slots unaggregate/aggregate guidelines

It's also good practice. It shouldn't be blocked and caller shouldn't need to wait for server to warm up the cache